### PR TITLE
Chronos: rename sparkdf_training_nyc_taxi.py and add example test

### PR DESCRIPTION
**Pull request by Rebekk**

### CODE REVIEW

File: python/chronos/dev/example/run-example-tests-pip-ray.sh [hunk 1]
```diff
@@ -83,12 +83,21 @@ sed -i 's/epochs=10/epochs=1/' "${BIGDL_ROOT}/python/chronos/example/quantizatio
 execute_ray_test quantization_tcnforecaster_nyc_taxi "${BIGDL_ROOT}/python/chronos/example/quantization/quantization_tcnforecaster_nyc_taxi.py"
 time7=$?
 
+if [ ! -f ~/.chronos/dataset/nyc_taxi/nyc_taxi_data.csv ]; then
+  wget -nv $FTP_URI/analytics-zoo-data/apps/nyc-taxi/nyc_taxi.csv -P ~/.chronos/dataset/nyc_taxi/
+  mv ~/.chronos/dataset/nyc_taxi/nyc_taxi.csv ~/.chronos/dataset/nyc_taxi/nyc_taxi_data.csv
+fi
+
+execute_ray_test sparkdf_training_nyc_taxi.py "${BIGDL_ROOT}/python/chronos/example/distributed/sparkdf_training_nyc_taxi.py"
```
Comment (TheaperDeng) [inline python/chronos/dev/example/run-example-tests-pip-ray.sh:91]: I think you need to set `--datadir ~/.chronos/dataset/nyc_taxi/nyc_taxi_data.csv`

Comment (Rebekk) [reply python/chronos/dev/example/run-example-tests-pip-ray.sh]: ok

**Comment by TheaperDeng** [comment]
is there any way to run this test before merging? @liangs6212

**Comment by liangs6212** [comment]
> is there any way to run this test before merging? @liangs6212

Sure, Just replace `Git_repo` and `Branch_Name`, a reference is given below.
http://10.112.231.51:18888/job/BigDL-NB-chronos-ExampleTests-py37-ray-spark3/262/parameters/

**Comment by TheaperDeng** [comment]
#4614 fixed a bug for this example, please test this PR after 4614 is merged

**Comment by Rebekk** [comment]
> 

ok

**Comment by liangs6212** [comment]
Jenkins Passed
http://10.112.231.51:18889/view/BigDL-PR-Validation/job/BigDL-Chronos-PR-Validation/395/
http://10.112.231.51:18889/job/BigDL-PRNV-chronos-Python-Spark-2.4-py37-Horovod/414/
http://10.112.231.51:18889/job/BigDL-PRVN-chronos-Python-Spark-3.1-py37-Horovod/442/

**Comment by TheaperDeng** [comment]
http://10.112.231.51:18888/job/BigDL-NB-chronos-ExampleTests-py37-ray/281/
http://10.112.231.51:18888/job/BigDL-NB-chronos-ExampleTests-py37-ray-spark3/273/

**Pull request merged**